spec/custom_matchers.rb

```diff
@@ -0,0 +1,12 @@
+RSpec::Matchers.define :belong_to do |expected|
+  match { |actual| (actual.to_a.uniq - expected.to_a.uniq).empty? }
+end
```

Comment:
Ah, yes. I realized this while writing json-importer tests, and was planning to add `:nrows` and `:ncols` too in the fray. Will change this soon.